src/untitled/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,7 @@
+package org.example;
+
+public class Main {
```

Comment:
Not sure if I miss something here, but I think this class is not used. If it's not used, I think it will be better if you remove this.